src/components/LeftSidebar/LeftSidebar.jsx

```diff
@@ -1,104 +1,35 @@
-import React, { Component } from "react";
+import React, { useState } from "react";
 import PropTypes from "prop-types";
-import Badge from "@material-ui/core/Badge";
+import { AiOutlineArrowRight } from "react-icons/ai";
+import { ImCross } from "react-icons/im";
 import "./LeftSidebar.scss";
-import NavItem from "../NavItem/NavItem";
-import AuthComponent from "../../security/authComponent";
-import { ROLES } from "../../constants/userConstants";
+import LeftSidebarNavigationSecurity from "./LeftSidebarNavigation/LeftSidebarNavigationSecurity";
 
-class LeftSidebar extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      class: "left-sidebar-closed",
-    };
-  }
-
-  render() {
-    const LEFT_SIDEBAR_OPENED = "left-sidebar-opened";
-    const LEFT_SIDEBAR_CLOSED = "left-sidebar-closed";
-    return (
-      <>
-        <div
-          id="open-close-zone"
-          className={`${this.state.class} d-flex justify-content-start`}
-          onClick={() => {
-            return this.state.class === LEFT_SIDEBAR_OPENED
-              ? this.setState({ class: LEFT_SIDEBAR_CLOSED })
-              : this.setState({ class: LEFT_SIDEBAR_OPENED });
-          }}
-          onKeyPress={() => {}}
-          role="button"
-          tabIndex="0"
+const LeftSidebar = ({ user, msgForRead }) => {
+  const [sidebarOpen, setSidebarOpen] = useState(false);
+  const handleSidebarToggle = () => {
+    setSidebarOpen(!sidebarOpen);
+  };
+  return (
+    <ul className={`sidebar-menu ${sidebarOpen ? "active" : ""}`}>
```

Comment:
empty  string? use && operator

Reply:
If i do so it will be "false" in className